xml/snmp_sysdescr.xml

```diff
@@ -5180,7 +5074,7 @@ Copyright (c) 1995-2005 by Cisco Systems
     <example>Rectifier SNMP Server</example>
     <param pos="0" name="os.vendor" value="Rectifier Technologies"/>
     <param pos="0" name="os.family" value="RTP Power Controller"/>
-    <param pos="0" name="os.device" value="General"/>
+    <param pos="0" name="os.device" value="Power Device"/>
```

Comment:
How does `Power Device` differ from `Power Management`. Which category would a managed APC power strip or UPS go into?  Should we consolidate?  Also, we should likely correct the capitalization in these other locations so as to capitalize the first letter in each word like you've used here.
```
ftp_banners.xml:    <param pos="0" name="os.device" value="Power device"/>
ftp_banners.xml:    <param pos="0" name="hw.device" value="Power device"/>
ftp_banners.xml:    <param pos="0" name="os.device" value="Power device"/>
ftp_banners.xml:    <param pos="0" name="hw.device" value="Power device"/>
http_servers.xml:    <param pos="0" name="hw.device" value="Power Management"/>
http_servers.xml:    <param pos="0" name="hw.device" value="Power Management"/>
http_wwwauth.xml:    <param pos="0" name="os.device" value="Power device"/>
telnet_banners.xml:    <param pos="0" name="hw.device" value="Power Management"/>
```